diff --git a/src/game/game_str.c b/src/game/game_str.c
index 013856e..043bda8 100644
--- a/src/game/game_str.c
+++ b/src/game/game_str.c
@@ -9,10 +9,10 @@
 
 /* -------------------------------------------------------------------------- */
 
-const char *game_str_mm_continue = "Continue";
-const char *game_str_mm_load = "Load Game";
-const char *game_str_mm_new = "New Game";
-const char *game_str_mm_quit = "Quit to OS";
+const char *game_str_mm_continue = "繼續";
+const char *game_str_mm_load = "讀取進度";
+const char *game_str_mm_new = "新遊戲";
+const char *game_str_mm_quit = "離開";
 
 const char *game_str_tbl_race[RACE_NUM + 1] = { "Human", "Mrrshan", "Silicoid", "Sakkra", "Psilon", "Alkari", "Klackon", "Bulrathi", "Meklar", "Darlok", "Random" };
 const char *game_str_tbl_races[RACE_NUM] = { "Humans", "Mrrshans", "Silicoids", "Sakkras", "Psilons", "Alkaris", "Klackons", "Bulrathis", "Meklars", "Darloks" };
